pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-headers.adoc

```diff
@@ -95,11 +102,7 @@ Because Envoy uses the same buffer to share data from the headers and the body,
 . Read the headers and save the necessary values in a variable.
 . Read the body.
 
-You can read the headers and then the body in both the response and request.
-
-You cannot modify headers after reading the body. Complete all header modification before reading the body.
-
-For example:
+You can read the headers and then the body in both the response and request. However, you cannot modify headers after reading the body. Complete all header modification before reading the body, for example:
```

Comment:
I added the comment in the code, but we could also add a short sentence here saying. If the body is going to be modified, it will be required to remove the incoming request content-length header value, since it will be no longer valid.